.github/workflows/helm-chart-lint.yml

```diff
@@ -1,18 +1,43 @@
 name: Lint Helm Charts
 
-on: pull_request
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - "helm/provisioner/**"
 
 jobs:
   lint-test:
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
         with:
           fetch-depth: 0
 
-      - name: Set up chart-testing
-        uses: helm/chart-testing-action@v2.1.0
+      - name: Set-up Helm
+        uses: azure/setup-helm@f382f75448129b3be48f8121b9857be18d815a82
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This requires setting up a secret through the Github UI, if that's true whose secret should we use?

Reply:
This secret is provided by GH Actions.